### PR TITLE
New c++ cheat sheet

### PR DESCRIPTION
**Pull request by jophab**
**What does your Instant Answer do?**
c++ reference
**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.cs.ccu.edu.tw/~damon/oop/,c++refcard.pdf
**Why did you choose this data source?**
Its good
**Are there any other alternative (better) data sources?**
Yes,may be
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
students
**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
no
**Are you having any problems? Do you need our help with anything?**
no
**Where did you hear about DuckDuckHack? (For first time contributors)**
From my friend
**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

-----
IA Page: https://duck.co/ia/view/cpp_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=C++ Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/cpp_cheat_sheet_mobile.png)

**Comment by jophab** [comment]
@moollaza the link given in beta is leading to c cheatsheet. Not c++ cheatsheet

**Comment by jophab** [comment]
@moollaza  @mintsoft  any more changes?

File: share/goodie/cheat_sheets/json/cpp.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
+ {
+    "id": "cpp_cheat_sheet",
+    "name": "C++",
+    "metadata": {
+        "sourceName": "CCU",
+        "sourceUrl": "http://www.cs.ccu.edu.tw/~damon/oop/,c++refcard.pdf"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/cpp.json]: Should there be a `,`in this url?

Comment (jophab) [inline share/goodie/cheat_sheets/json/cpp.json]: @mintsoft yes there is..
please verify

Comment (moollaza) [inline share/goodie/cheat_sheets/json/cpp.json]: @mintsoft @jophab that’s certainly a very odd URL!

**Comment by moollaza** [comment]
@jophab this LGTM :+1:

**Pull request merged**